cmd/sops/subcommand/exec/exec.go

```diff
@@ -83,15 +84,18 @@ func ExecWithEnv(opts ExecOpts) {
 	}
 
 	env := os.Environ()
-	lines := bytes.Split(opts.Plaintext, []byte("\n"))
-	for _, line := range lines {
-		if len(line) == 0 {
-			continue
-		}
-		if line[0] == '#' {
+	store := dotenv.Store{}
+
+	branches, err := store.LoadPlainFile(opts.Plaintext)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Could we include a more descriptive error message alongside the error value?

Reply:
I agree in spirit, but that seems to deviate from the norm established by the other functions in this file. Any suggestions?

Reply:
Hmm, yeah I see your point. In that case, let's just leave it as-is for now.

Reply:
Sounds good!